docs/topics/spiders.rst

```diff
@@ -145,6 +145,41 @@ scrapy.Spider
        :param kwargs: keyword arguments passed to the :meth:`__init__` method
        :type kwargs: dict
 
+   .. classmethod:: update_settings(settings)
+
+       The ``update_settings()`` method is used to modify the spider's settings
+       and is called during initialization of a spider instance.
+
+       It takes a :class:`~scrapy.settings.Settings` object as a parameter and
+       can add or updates the spider's configuration values. This method is a class method,
+       meaning that it is called on the :class:`~scrapy.Spider` class and allows all instances
+       of the spider to share the same configuration.
+
+       One of the main advantages of ``update_settings()`` is that it allows
+       you to dynamically add, remove or change settings based on spider arguments
+       or other external factors.
```

Comment:
I just realized I got it wrong here, you cannot take spider arguments into account, because those are set in the `__init__`, and as you say here this is a class method. You can still edit settings based on other settings or external factors, though, so that part is indeed true.

Reply:
Done. The last commit updated this part